### PR TITLE
Split FoolSlide Extension

### PR DESCRIPTION
**Pull request by Riztard**

### CODE REVIEW

File: multisrc/overrides/foolslide/hentaicafe/src/HentaiCafe.kt [hunk 1]
```diff
@@ -1,22 +1,21 @@
...
 import org.jsoup.nodes.Document
 import org.jsoup.nodes.Element
+import eu.kanade.tachiyomi.network.asObservable
+import eu.kanade.tachiyomi.source.model.Filter
+import eu.kanade.tachiyomi.source.model.MangasPage
+import eu.kanade.tachiyomi.util.asJsoup
 import rx.Observable
 import java.net.URLEncoder
 
-@Nsfw
```
Comment (arkon) [inline multisrc/overrides/foolslide/hentaicafe/src/HentaiCafe.kt:19]: Why removing this?

Comment (Riztard) [reply multisrc/overrides/foolslide/hentaicafe/src/HentaiCafe.kt:19]: already add nsfw in generator?

Comment (arkon) [reply multisrc/overrides/foolslide/hentaicafe/src/HentaiCafe.kt:19]: That only affects the gradle file and generated class. If you're overriding the class, you still need to have it explicitely.

Comment (Riztard) [reply multisrc/overrides/foolslide/hentaicafe/src/HentaiCafe.kt:19]: naruhodo

**Pull request merged**